G.A.M.M.A/modpack_addons/A Few Random Fixes/gamedata/scripts/a_blood_pool.script

```diff
@@ -0,0 +1,3 @@
+if z_npc_footsteps then
+  AddScriptCallback("npc_on_foot_step")
+end
```

Comment:
I forgot you disabled the Blood Pool mod (I still had it enabled from before) so you might not care about this particular change. But the problem originates here:

![2](https://github.com/Grokitach/Stalker_GAMMA/assets/152332990/423bb43a-0f0c-41e1-8e06-3bae3136ad2f)

If `z_npc_footsteps` exists, it calls `RegisterScriptCallback("npc_on_foot_step", npc_on_foot_step)` without calling `AddScriptCallback("npc_on_foot_step")` first. So the call is invalid. And it looks like the code that doesn't run as a result is responsible for placing bloody footprints:

![3](https://github.com/Grokitach/Stalker_GAMMA/assets/152332990/e1103fb6-4242-43c8-8398-9b1d5f58b041)

`z_npc_footsteps.script` is part of the mod "9- Exo Servomotor Sounds". So the bug exists as long as that mod is enabled.

My fix just calls `AddScriptCallback("npc_on_foot_step")` beforehand to make sure it's always valid.